src/libraries/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.csproj

```diff
@@ -9,7 +9,7 @@
   <PropertyGroup>
     <AssemblyVersion Condition="'$(TargetFramework)' == 'netstandard2.0'">4.1.0.0</AssemblyVersion>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetsUnix)' != 'true'">
+  <PropertyGroup Condition="'$(TargetsUnix)' != 'true' or '$(TargetsBrowser)' == 'true'">
```

Comment:
Does the browser target claim to be Unix?  If not, the or seems redundant.

Reply:
It does right now, but that will be changed with https://github.com/dotnet/runtime/pull/37944. For now this is needed.